GCRCatalogs/catalog_configs/dc2_object_run2.1i_dr1b_with_photoz_tract3830.yaml

```diff
@@ -0,0 +1,15 @@
+subclass_name: composite.CompositeReader
```

Comment:
Why not using the `based_on` technique here to restrict the previous catalog to one tract ?

Reply:
because the tract specification needs to be pass onto each of the sub-catalogs. If we use `based_on`, we still need to write out the whole `catalogs` list, so not much save here.